From b542742ef9eb66602c0e205a27552b1140e85eb7 Mon Sep 17 00:00:00 2001
From: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Date: Tue, 8 Mar 2016 14:16:01 -0600
Subject: [PATCH] mk: add symbolic link to dpdk combined lib

In order to have the dpdk files in the correct
packages, it's necessary to add a symbolic link
to dpdk combined library, otherwise the dpdk lib will be
installed in a wrong package.

Signed-off-by: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
---
 mk/rte.sharelib.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mk/rte.sharelib.mk b/mk/rte.sharelib.mk
index 70c49a8..45b5153 100644
--- a/mk/rte.sharelib.mk
+++ b/mk/rte.sharelib.mk
@@ -36,7 +36,7 @@ VPATH += $(SRCDIR)
 
 ifeq ($(CONFIG_RTE_BUILD_COMBINE_LIBS),y)
 ifeq ($(CONFIG_RTE_BUILD_SHARED_LIB),y)
-LIB_ONE := lib$(RTE_LIBNAME).so
+LIB_ONE := lib$(RTE_LIBNAME).so.2.2
 else
 LIB_ONE := lib$(RTE_LIBNAME).a
 endif
@@ -84,6 +84,7 @@ $(LIB_ONE): FORCE
 	@[ -d $(dir $@) ] || mkdir -p $(dir $@)
 	@$(SRCDIR)/scripts/merge-maps.sh > $(COMBINED_MAP)
 	$(O_TO_S_DO)
+	$(Q)ln -s -f $(LIB_ONE) $(basename $(basename $(RTE_OUTPUT)/lib/$(LIB_ONE)))
 else
 $(LIB_ONE): FORCE
 	@[ -d $(dir $@) ] || mkdir -p $(dir $@)
-- 
2.6.4.442.g545299f.dirty

